ibm/service/kubernetes/resource_ibm_container_vpc_worker_pool.go

```diff
@@ -232,6 +240,33 @@ func ResourceIBMContainerVPCWorkerPoolValidator() *validate.ResourceValidator {
 
 func resourceIBMContainerVpcWorkerPoolCreate(d *schema.ResourceData, meta interface{}) error {
 
+	if ioc, ok := d.GetOk("import_on_create"); ok && ioc.(bool) {
+		if cluster, ok := d.GetOk("cluster"); ok {
+			clusterID := cluster.(string)
+			log.Printf("Importing workerpool from cluster %s", clusterID)
+
+			//read to get ID for default and d.Set!
+			wpClient, err := meta.(conns.ClientSession).VpcContainerAPI()
+			if err != nil {
+				return err
+			}
```

Comment:
this client init code can be shared between the import_on_create and the regular create code path